### PR TITLE
Questions page guess quiz

### PR DESCRIPTION
**Pull request by Maianki**
## Added Quiz Question Page

Screenshot (600)

## Live Link
https://quiz-questions-page.netlify.app/quiz-pages/guessplayer-quiz/question-1.html

### CODE REVIEW

**Review by vrx29 — APPROVED** [review]
looks good to me.

File: styles/quesiton.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (venkateshwarans) [inline styles/quesiton.css]: typo in the file name.

**Review by venkateshwarans — APPROVED** [review]
LGTM. Good work.

**Comment by Maianki** [comment]
> looks good to me.

Thanks for reviewing.

File: styles/quesiton.css (hunk 1 above)
Comment (Maianki) [reply styles/quesiton.css]: Thanks for pointing this out. Will fix this.

**Comment by Maianki** [comment]
> LGTM. Good work.

Thanks for reviewing.

**Comment by Maianki** [comment]
Fixed the typo in ```question.css```. Merging PR.

**Pull request merged**